### PR TITLE
adds missing lerna commands from lernajs.io

### PR DESCRIPTION
**Pull request by dakebl**
Answering the issue I raised about missing commands (#59)

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -135,6 +135,72 @@ <h3><a href="#command-bootstrap">
...
+          </p>
+          <p>
+            <strong>
+              Location will default to the first configured package
+              location.
+            </strong>                   
+          </p>
+        </section>
+
+        <section class="command" id="command-ls">
```
Comment (doug-wade) [inline index.html:160]: id="command-add"

File: index.html [hunk 2]
```diff
@@ -135,6 +135,72 @@ <h3><a href="#command-bootstrap">
...
+          <h3><a href="#command-ls">
```
Comment (doug-wade) [inline index.html:161]: href="#command-add"

File: index.html [hunk 3]
```diff
@@ -135,6 +135,72 @@ <h3><a href="#command-bootstrap">
...
+            <code>lerna add &lt;package&gt;</code>
+          </a></h3>
+
+          <p>
+            Add a dependency to matched packages.
+          </p>
+          <p>
+            <p>
+              <code>
+                # Install module-1 to module-2 <br/>
```
Comment (doug-wade) [inline index.html:171]: Add module-1 as a dependency of module-2 (install is one-time, adding as a dependency is permanent)

File: index.html [hunk 4]
```diff
@@ -135,6 +135,72 @@ <h3><a href="#command-bootstrap">
...
+                lerna add module-1 --scope=module-2
+              </code>
+            </p>
+            <p>
+              <code>
+                # Install module-1 in all modules except module-1 <br/>
```
Comment (doug-wade) [inline index.html:177]: This one and the one following it seem identical at first glance.  Please add some disambiguation.

File: index.html [hunk 5]
```diff
@@ -184,17 +250,53 @@ <h3><a href="#command-changed">
...
-            <code>lerna diff [package?]</code>
+            <code>lerna diff &lt;package&gt;</code>
           </a></h3>
 
           <p>
             Diff all packages or a single package since the last release.
           </p>
         </section>
 
+        <section class="command" id="command-ls">
```
Comment (doug-wade) [inline index.html:261]: id="command-version"

File: index.html [hunk 6]
```diff
@@ -184,17 +250,53 @@ <h3><a href="#command-changed">
...
+          <h3><a href="#command-ls">
```
Comment (doug-wade) [inline index.html:262]: href="#command-version"

File: index.html [hunk 7]
```diff
@@ -184,17 +250,53 @@ <h3><a href="#command-changed">
...
+            <code>lerna version &lt;version&gt;</code>
+          </a></h3>
+
+          <p>
+            Bump the version of packages changed since the last release.
+          </p>
+          <p>
+            <code>lerna version 1.0.1 # explicit </code><br/>
+            <code>lerna version patch # semver keyword </code><br/>
```
Comment (doug-wade) [inline index.html:271]: \# semver keyword, as enumerated below

File: index.html [hunk 8]
```diff
@@ -203,6 +305,20 @@ <h3><a href="#command-run">
           </p>
         </section>
 
+        <section class="command" id="command-ls">
+          <h3><a href="#command-ls">
```
Comment (doug-wade) [inline index.html:309]: href="#command-exec"

File: index.html [hunk 9]
```diff
@@ -203,6 +305,20 @@ <h3><a href="#command-run">
           </p>
         </section>
 
+        <section class="command" id="command-ls">
```
Comment (doug-wade) [inline index.html:308]: id="command-exec"

File: index.html [hunk 10]
```diff
@@ -203,6 +305,20 @@ <h3><a href="#command-run">
...
+            Run an arbitrary command in each package. 
+          </p>
+          <p>
+            <code>
+              lerna exec -- rm -rf ./node_modules
+            </code>
+          </p>
+        </section>
+
         <section class="command" id="command-ls">
```
Comment (doug-wade) [inline index.html:322]: This seems like some mistaken copy-pasta

**Review by doug-wade — CHANGES_REQUESTED** [review]

**Comment by JamesHenry** [comment]
Hi @dakebl 👋 

Thanks a lot for taking the initiative to contribute this PR!

You may or may not know that lerna is now under the stewardship of Nrwl (announcement here https://github.com/lerna/lerna/issues/3121), a company with a long history of not just producing valuable open-source software (OSS), but also backing others (at the time of writing, Nrwl has donated over $50,000 to OSS it _hasn't_ created, see https://opencollective.com/nx for full details).

Quite simply, Nrwl ❤️  OSS, and is committed to making lerna the best it can be. We use it ourselves.

We hope you will continue to be a part of this community as we look to take things forward from here!

Please see https://github.com/lerna/lerna/discussions/3140 for more details on our plans for 2022.

We have just pushed out a new version of the website, which we will continue to iterate on, we hope you like it and find it useful!

Feel free to open a new issue on here if you spot anything that needs fixing or improving, but for now this PR is no longer applicable to the current site so we are closing it.

Many thanks 🙏

**Pull request closed without merging**